app/src/main/AndroidManifest.xml

```diff
@@ -61,6 +61,15 @@
             android:name=".settings.SettingsActivity"
             android:exported="true" />
 
+        <activity-alias android:name=".settings.SettingsLink"
+                android:targetActivity=".settings.SettingsActivity"
+                android:label="@string/backup">
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN" />
+                <category android:name="android.intent.category.LAUNCHER" />
+            </intent-filter>
```

Comment:
Why does it need an alias? Could the intent filter not be added to the activity above?

Reply:
Yeah, good point, it could be. I was initially looking at something else which had a different label, but here there's no label so it's fine.